modules/about-gitops.adoc

```diff
@@ -0,0 +1,12 @@
+// Module is included in the following assemblies:
+//
+// * openshift-docs/cicd/gitops/understanding-openshift-gitops.adoc
+
+[id="about-gitops_{context}"]
+= About GitOps
+
+GitOps is a declarative way to implement continuous deployment for cloud native applications. You can use GitOps to create repeatable processes for managing OpenShift Container Platform clusters and applications across multi-cluster Kubernetes environments. GitOps handles and automates complex deployments at a fast pace, saving time during deployment and release cycles. Its workflow pushes an application through all its stages, development, testing, staging, and production. GitOps either deploys a new application or updates an existing one, so you only need to update the repository. It handles and automates everything else.
+
+GitOps is a set of practices that use Git pull requests to manage infrastructure and application configurations. Its Git repository is the only source of truth for system and application configuration. This Git repository contains a declarative description of the infrastructure you need in your specified environment and contains an automated process to make your environment match the described state. Also, it contains the entire state of the system so that the trail of changes to the system state are visible and auditable. By using GitOps, you resolve the issues of infrastructure and application configuration sprawl.
```

Comment:
s/ Its Git repository / In GitOps, the Git repository

Reply:
Might want to explain how GitOps differs from [Infrastructure-as-Code](https://docs.microsoft.com/en-us/azure/devops/learn/what-is-infrastructure-as-code), which also resolves the issues of infrastructure and application configuration sprawl?